types/noble-mac/tslint.json

```diff
@@ -0,0 +1,79 @@
+{
+    "extends": "dtslint/dt.json",
+    "rules": {
+        "adjacent-overload-signatures": false,
```

Comment:
Whoa! That's a lot of lint exceptions for a new type definition! Generally we don't like to take new types with exceptions like these (the exceptions exist to opt-out old types until someone gets around to updating them). Could you not disable all these rules and fix any of the resulting issues?

Reply:
Thanks for your comment. I will check and change this in the next days.

Reply:
Alright, I reduced the linter config exceptions to a minimum and fixed the resulting issues.